distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc

```diff
@@ -18,9 +18,7 @@ It is recommended to apply the following security mitigations to the ${branding}
 
 |JMX
 |tampering, information disclosure, and unauthorized access
-a|* Remove `-Dcom.sun.management.jmxremote` from `<${branding}_HOME>/bin/karaf`. +
-* Disable ${branding}'s JMX management `rmiRegistryPort` and `rmiServerPort` (`1099`, `44444`) by removing these entries from `etc/org.apache.karaf.management.cfg`. +
-* Uninstall the management bundle using the command line console: `uninstall management`. +
+a|* Stop the management feature using the command line console: `feature:stop management`. +
```

Comment:
Probably don't _need_ it... But I thought it should stay in there if for any reason someone removed the -XX:+DisableAttachMechanism arg, then they can still stop the management feature while running as a hardening step.